assets/js/blocks/collection-filters/save.tsx

```diff
@@ -0,0 +1,11 @@
+/**
+ * External dependencies
+ */
+import { useBlockProps, useInnerBlocksProps } from '@wordpress/block-editor';
+
+export default function save() {
+	const blockProps = useBlockProps.save();
+	const innerBlockProps = useInnerBlocksProps.save( blockProps );
+
+	return <div { ...innerBlockProps } />;
```

Comment:
My usual nitpicks… what about using a different tag? Personally, I find `nav` pretty apt as the container of various filters. To the very least, I would say `section` is appropriate.

Reply:
Agree, let's use `<nav>` for now.

@sunyatasattva I think merchants may have different preferences on the tags they want to use, should we add an option here to let merchants control which tag to use?

Reply:
Moved this to the parent issue. https://github.com/woocommerce/woocommerce-blocks/issues/10711#issuecomment-1718771574